src/components/CreateScenarioButton/CreateScenarioButtonHook.js

```diff
@@ -54,37 +59,27 @@ export const useCreateScenarioButton = ({ disabled, onScenarioCreated }) => {
   }, [solution?.data?.runTemplates, workspaceData?.solution?.runTemplateFilter, t, clone]);
 
   const filteredDatasets = useMemo(() => {
-    const datasetFilter = workspaceData?.webApp?.options?.datasetFilter;
-    if (!datasetFilter) {
-      return datasets;
-    }
-
-    if (!Array.isArray(datasetFilter) || !datasetFilter.length) {
-      console.warn(logsLabels.warning.datasetFilter.emptyOrNotArray);
-      return datasets;
-    }
+    const getMainDatasets = () => datasets.filter((dataset) => dataset.main === true);
+    if (!datasetsFilter) return getMainDatasets();
 
     const result = [];
-    datasetFilter.forEach((dsetFilter) => {
-      if (typeof dsetFilter !== 'string') {
-        console.warn(logsLabels.warning.datasetFilter.getNotAString(dsetFilter));
-      } else {
-        const filteredDataset = datasets.find((dset) => dset.id === dsetFilter);
-        if (!filteredDataset) {
-          console.warn(logsLabels.warning.datasetFilter.getDatasetNotFoundForFilter(dsetFilter));
-        } else {
-          result.push(filteredDataset);
-        }
+    datasetsFilter.forEach((filterItem) => {
+      if (typeof filterItem !== 'string')
+        console.warn(`Ignoring dataset filter entry ${filterItem} because it is not a string`);
+      else {
+        const filteredDataset = datasets.find((dset) => dset.id === filterItem);
+        if (!filteredDataset) console.warn(`No dataset found for filter entry ${filterItem}`);
+        else result.push(filteredDataset);
       }
     });
 
-    if (!result.length) {
-      console.warn(logsLabels.warning.datasetFilter.noDatasetFound);
-      return datasets;
+    if (datasetsFilter.length > 0 && result.length === 0) {
```

Comment:
No it isn't, because an empty filter list is a valid use case, we don't want to log a warning in this case, even though the results list will be empty